packages/block-library/src/pullquote/block.json

```diff
@@ -60,6 +60,12 @@
 				"style": true,
 				"width": true
 			}
+		},
+		"__experimentalStyle": {
+			"typography": {
+				"fontSize": "1.5em",
```

Comment:
Can you clarify why this font size was chosen? Only because it is in between the two current values?
(I do not know why there were two font sizes in the first place)

Reply:
Yep, I chose that font size because it is in the middle of the previous 2 sizes used here, with the intention of fixing the problem and causing fewer possible changes on sites using already using the block.